### DISCUSSION TITLE
Vagter vises ikke på "Alle dage" vagtoverblik

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#10] [Bugs] [answered]
Vagter er forvundet fra "Alle dage" vagtoverblik:

![image](https://github.com/user-attachments/assets/7190d882-07d7-4e3f-afb2-8b0f459b0719)

![image](https://github.com/user-attachments/assets/21769a00-ce1f-4e92-8df5-bbed2c0780e0)

**Comment by antonlund18** [comment, marked as answer]
Det er en bug fra da jeg tilføjede filtre herinde. Jeg fikser det. Indtil da, kan du sætte et navn på holdet, og så vil det dukke op i overblikket.
